modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -208,30 +222,30 @@ function widget( $args, $instance ) {
 		 */
 		?>
 		<script type="text/javascript">
-			jQuery( document ).ready( function( $ ) {
-				var actionUrl      = <?php echo json_encode( home_url( '/' ) ); ?>,
+			jQuery(document).ready(function ($) {
+				var actionUrl = <?php echo json_encode( home_url( '/' ) ); ?>,
 					orderByDefault = <?php echo json_encode( $orderby ); ?>,
-					orderDefault   = <?php echo json_encode( $order ); ?>,
-					widgetId       = <?php echo json_encode( $this->id ); ?>,
-					currentSearch  = <?php echo json_encode( isset( $_GET['s'] ) ? $_GET['s'] : '' ); ?>
+					orderDefault = <?php echo json_encode( $order ); ?>,
+					widgetId = <?php echo json_encode( $this->id ); ?>,
+					currentSearch = <?php echo json_encode( isset( $_GET['s'] ) ? $_GET['s'] : '' ); ?>
 
 				var container = $('#' + widgetId);
 				var form = container.find('.jetpack-search-form form');
-				var orderBy = form.find( 'input[name=orderby]');
-				var order = form.find( 'input[name=order]');
+				var orderBy = form.find('input[name=orderby]');
+				var order = form.find('input[name=order]');
 				orderBy.val(orderByDefault);
 				order.val(orderDefault);
 
-				container.find( '.jetpack-search-sort' ).change( function( event ) {
-					var values  = event.target.value.split( '|' );
-					orderBy.val( values[0] );
-					order.val( values[1] );
+				container.find('.jetpack-search-sort').change(function (event) {
+					var values = event.target.value.split('|');
+					orderBy.val(values[0]);
+					order.val(values[1]);
```

Comment:
I can see that in some places your editor removes spaces inside function calls, isn't that against WordPress coding standards?

Reply:
I know that's the case for PHP but I was thinking it was different for JavaScript. Turns out I was wrong:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls

I'll fix it. Thanks!